crawl-ref/source/dat/descript/de/monsters.txt

```diff
@@ -854,7 +854,7 @@ Waffen ausgestattet ist und diese geschickt einzusetzen weiß.
 chaos spawn
 
 Ein Wesen des reinen Chaos, dessen Gestalt sich ständig verändert, das Augen,
-Münder, Klauen und Tentakeln wächst und dann verliert. Es verdirbt alles, was es
+Münder, Klauen und Tentakel wächst und dann verliert. Es verdirbt alles, was es
```

Comment:
This should be:
Ein Wesen aus reinem Chaos, dessen Gestalt sich ununterbrochen verändert. Ihm wachsen neue Augen, Münder, Klauen und Tentakel und verschwinden direkt wieder. Es korrumpiert alles, was es berüht.